.travis.yml

```diff
@@ -5,7 +5,7 @@ rvm:
 services:
 - redis-server
 script:
-- bundle exec rspec
+- bundle exec rspec --format d
```

Comment:
Yeah it's fine with me, just wanted to point it out in case it was an oversight from debugging or anything.